docs/update-from-client-side-helpers-to-server-side.md

```diff
@@ -0,0 +1,229 @@
+<br/><br/>
+> 
+> **Know before you code**
+>  
+> [How to Setup a Developer Account](https://www.youtube.com/watch?v=O_9G722SpXQ&t=72s)
+> <br/><br/>
+
+# What do I need to change?
+
+If you are reading this page, we expect that you already have a client-side PayPal Button integration in place.  That said, several steps are required to get you up and running with the recommended approach. 
+
+Specifically:
+
+1. Setup a server-side integration with the [PayPal REST API's](https://developer.paypal.com/api/rest/).
+
+1. `createOrder` JavaScript callback must be changed to use your server-side to create and return an Order ID using the PayPal backend API's.
+
+1. `onApprove` JavaScript callback must be changed to use your server-side to complete transactions using the PayPal backend API's.
+
+<br/><br/>
+
+# 1. Setup a server-side integration with the PayPal REST API's
+
+PayPal REST API's follow a common integration approach used by other RESTful API's out there in the world.  For PayPal, you will minimally need to do the following:
+
+1. Setup a PayPal developer account if you have not already.
+
+1. Create a PayPal REST application and obtain a client + secret
+
+1. Use the client/secret combination to obtain an oAuth access token
+
+1. Use the oAuth access token to directly make REST calls to the PayPal Order API's -- which you will need for this document.
+
+<br/><br/>
+
+# 2. Change your `createOrder()` callback
+
+Your current integration may likely be using the client-side order create functions.  You will need to replace this code to call your server side instead.  Examples on before and after are below.
+
+## Current state of client-side integration behavior
+
+With the client-side integration pattern, the SDK code would take care of calling the v2 Orders API on your behalf. It would also handle the authorization for you.
+
+```js
+createOrder(data, actions) {
+  return actions.order.create({
+    purchase_units: [
+      {
+          amount: {
+            value: "88.44",
+          },
+      }],
+  });
+}
+```
+### Changes needed to migrate to using server-side integration pattern
+
+To simplify the integration of your e-commerce website with the PayPal v2 Orders API, you can move the order creation process to your server-side. The following steps are required to create an order on the server-side:
+
+1. Obtain an access token to use for PayPal backend API calls. This [video tutorial](https://www.youtube.com/watch?v=HOkkbGSxmp4&t=113s) can walk you through the steps. 
+2. Pass necessary checkout info from the browser client to your server-side API endpoint.
+3. Call the PayPal Orders API from your server-side code and return the order ID in your createOrder() callback. 
+
+## New state of client + server integration behavior
+
+```js
+createOrder: function (data, actions) {
+  return (
+      // 1. send your cart info to your server side to create a PayPal Order.
```

Comment:
This is minor, but I think the numbering in the code comments can be confusing. At first I thought it tied to the above list but that's not always the case. What do you think about removing them?
```suggestion
      // send your cart info to your server side to create a PayPal Order.
```